website/docs/reference/project-configs/require-dbt-version.md

```diff
@@ -16,19 +15,23 @@ require-dbt-version: version-range | [version-range]
 
 ## Definition
 
-You can use `require-dbt-version` to restrict your project to only work with a range of dbt versions. 
+You can use `require-dbt-version` to restrict your project to only work with a range of dbt versions.
 
-When you set this configuration, dbt sends a helpful error message for any user who attempts to run the project with an unsupported version of dbt. This can be useful for package maintainers (such as [dbt-utils](https://github.com/dbt-labs/dbt-utils)) to ensure that users' dbt version is compatible with the package. Setting this configuration might also help your whole team remain synchronized on the same version of dbt for local development, to avoid compatibility issues from changed behaviour.
+When you set this configuration:
+- dbt issues error messages for packages on the [dbt Packages hub](https://hub.getdbt.com/) that specify a `require_dbt_version` that doesn't match. 
+- This can be useful for package maintainers (such as [dbt-utils](https://github.com/dbt-labs/dbt-utils)) to ensure that users' dbt version is compatible with the package. 
+- Signals [compatibility with <Constant name="fusion_engine"/>](#fusion-compatibility) (`2.0.0` and higher).
+- It might also help your whole team remain synchronized on the same version of dbt for local development, to avoid compatibility issues from changed behavior.
 
-If this configuration is not specified, no version check will occur.
+You should pin to a major release. See [pin to a range](#pin-to-a-range) for more details. If this configuration isn't specified, no version check will occur.
```

Comment:
Based on the conversations I overheard, I thought this was a requirement (correct me if I'm wrong)
```suggestion
You must pin to a major release. See [pin to a range](#pin-to-a-range) for more details. If this configuration isn't specified, no version check will occur.
```